docs/sandbox/VehicleParking.md

```diff
@@ -32,75 +27,292 @@ All updaters have the following parameters in common:
 - `type`: this needs to be `"vehicle-parking"`
 - `feedId`: this is used as a "prefix" for park ids, entrance ids and sometimes also for tags.
 
-<b>To use HSL park updater:</b>
+## HSL Park and Ride
 
-```
+<!-- hsl-park BEGIN -->
+<!-- NOTE! This section is auto-generated. Do not change, change doc in code instead. -->
+
+| Config Parameter                |     Type    | Summary                                      |  Req./Opt. | Default Value | Since |
+|---------------------------------|:-----------:|----------------------------------------------|:----------:|---------------|:-----:|
+| type = "VEHICLE_PARKING"        |    `enum`   | The type of the updater.                     | *Required* |               |   na  |
+| facilitiesFrequencySec          |  `integer`  | How often the facilities should be updated.  | *Optional* | `3600`        |  2.2  |
+| facilitiesUrl                   |   `string`  | URL of the facilities.                       | *Optional* |               |  2.2  |
+| [feedId](#u__2__feedId)         |   `string`  | The name of the data source.                 | *Optional* |               |  2.2  |
+| hubsUrl                         |   `string`  | Hubs URL                                     | *Optional* |               |  2.2  |
+| [sourceType](#u__2__sourceType) |    `enum`   | The source of the vehicle updates.           | *Required* |               |  2.2  |
+| [timeZone](#u__2__timeZone)     | `time-zone` | The time zone of the feed.                   | *Optional* |               |  2.2  |
+| utilizationsFrequencySec        |  `integer`  | How often the utilization should be updated. | *Optional* | `600`         |  2.2  |
+| utilizationsUrl                 |   `string`  | URL of the utilization data.                 | *Optional* |               |  2.2  |
+
+
+#### Details
+
+<h4 id="u__2__feedId">feedId</h4>
+
+**Since version:** `2.2` ∙ **Type:** `string` ∙ **Cardinality:** `Optional`  \
+**Path:** /updaters/[2] 
```

Comment:
What does this path thing mean here?

Reply:
It's the JSON path in the config file. I must admit that for arrays it's  bit confusing. Maybe writing `/updaters/[]` is better.